extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/OperationType.scala

```diff
@@ -21,7 +21,7 @@ object OperationType extends Enumeration {
 
   type OperationType = Value
 
-  val ALTERDATABASE, ALTERDATABASE_LOCATION, ALTERTABLE_ADDCOLS, ALTERTABLE_ADDPARTS,
+  val ALTERDATABASE, ALTERDATABASE_LOCATION, ALTERTABLE, ALTERTABLE_ADDCOLS, ALTERTABLE_ADDPARTS,
```

Comment:
Prevent Introducing new OperationType if there are not enough reasons.
Authz plugin follows the Ranger's Hive policy style and  the operation type refers to  upstream Ranger Hive plugin's `RangerHiveOperationType`(https://github.com/apache/ranger/blob/ranger-2.3/hive-agent/src/test/java/org/apache/ranger/services/hive/RangerHiveOperationType.java)

Reply:
So, what should I do for such special command?

Reply:
Just use query?

Reply:
Just reuse other existed Operation Type. After all , it's only used inside the privilege parsing, not directly affects the final result.